gse-spi/src/test/java/com/powsybl/spi/PowsyblLogoTest.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.gse.spi;
+
+import org.junit.Test;
+
+/**
+ * @author Jon Harper <jon.harper at rte-france.com>
+ */
+public class PowsyblLogoTest {
+
+    @Test
+    public void testPowsyblLogo() {
+        //The svg is read in the constructor
+        new PowsyblLogo();
```

Comment:
Does this code really test / assert something? Do you expect it doesn’t throw any exception?

Reply:
The previous and new code throw ClassCastExceptions if the svg doesn't have a particular structure.